ext/odbc/tests/odbc_primarykeys_001.phpt

```diff
@@ -0,0 +1,48 @@
+--TEST--
+odbc_primarykeys(): Basic test for odbc_primarykeys()
+--EXTENSIONS--
+odbc
+--SKIPIF--
+<?php
+include 'skipif.inc';
+
+$conn = odbc_connect($dsn, $user, $pass);
+$result = @odbc_exec($conn, "SELECT @@Version");
+if ($result) {
+    $array = odbc_fetch_array($result);
+    $info = (string) reset($array);
+    if (str_contains($info, "Microsoft SQL Server")) {
+       echo "skip Doesn't work with MS SQL";
+    }
+}
+
+?>
+--FILE--
+<?php
+
+include 'config.inc';
+
+$conn = odbc_connect($dsn, $user, $pass);
+odbc_exec($conn, 'CREATE DATABASE PrimarykeysTest');
+odbc_exec($conn, 'CREATE TABLE primarykeys (test INT NOT NULL)');
+
+$res = odbc_primarykeys($conn, "", "PrimarykeysTest", "primarykeys");
+var_dump(odbc_fetch_array($res));
+odbc_free_result($res);
+
+odbc_exec($conn, 'ALTER TABLE primarykeys ADD CONSTRAINT primarykeys_pk PRIMARY KEY (test)');
+
+$res = odbc_primarykeys($conn, "", "PrimarykeysTest", "primarykeys");
+var_dump(odbc_fetch_array($res));
+odbc_free_result($res);
+
+?>
+--CLEAN--
+<?php
+require 'config.inc';
+$conn = odbc_connect($dsn, $user, $pass);
+odbc_exec($conn, 'DROP TABLE primarykeys');
+odbc_exec($conn, 'DROP DATABASE PrimarykeysTest');
+?>
+--EXPECT--
+bool(false)
```

Comment:
Aren't there values missing here? There are 2 var dumps but only one line of output?

Reply:
Finally, I managed to make this test work!

Previously, I had an suspicion that the `CREATE DATABASE` commands in the test directory are in vain (although I already removed most of them, that ones which were unnecessary), since the adjacent `CREATE TABLE` commands don't automatically use the new database, so the new tables end up being in the default database...

And this problem was crucial in this test, since one have to know the database name to fetch the primary key... So as soon as I added the `USE PrimarykeysTest` command, the test was fixed. :)